src/app/WriteHandler.h

```diff
@@ -112,6 +112,11 @@ class WriteHandler : public Messaging::ExchangeDelegate
         return mACLCheckCache.HasValue() && mACLCheckCache.Value() == aToken;
     }
 
+    bool IsConflictWrite(const ConcreteAttributePath & aPath)
```

Comment:
Should probably be called something like "IsCurrentlyChunkedWritePath" or something like that?